yt_dlp/extractor/txxx.py

```diff
@@ -371,6 +389,7 @@ def _real_extract(self, url):
             'like_count': int_or_none(traverse_obj(video_info, ('video', 'statistics', 'likes'))),
             'dislike_count': int_or_none(traverse_obj(video_info, ('video', 'statistics', 'dislikes'))),
             'age_limit': 18,
+            'thumbnail': traverse_obj(video_info, ('video', 'thumbsrc')),
```

Comment:
maybe we should use `url_or_none` just in case? Need to import it from `utils`
```suggestion
            'thumbnail': traverse_obj(video_info, ('video', 'thumbsrc', {url_or_none})),
```